environment.yml

```diff
@@ -4,17 +4,18 @@ channels:
   - conda-forge
 dependencies:
   - astropy>=4.0
-  - h5py
+  - astropy-healpix
   - line_profiler
   - mpi4py>=3.0.0
   - numpy>=1.15
   - pip
   - pre-commit
   - psutil
   - pypandoc
+  - python-casacore>=3.1.0
   - pytest
   - pytest-cov
-  - pyuvdata>=2.1.3
+  - pyuvdata>=2.1.5
```

Comment:
Maybe I should make this be >=2.2.2 since that's required for `write_ms` and this is the environment we provide for developers.

Reply:
If that's the version of pyuvdata we want to work/build against I am totally fine with that. It just requires bumping the required version in our next release.

Reply:
So the situation is that we require pyuvdata>=2.1.5, which I updated in the setup.py and in the CI yamls. But to use the `write_ms` functionality, we optionally require python-casacore and pyuvdata>=2.2.2. Since this yaml is the one we provide for developers and includes all the optional dependencies,  I think it should list pyuvdata>=2.2.2, but I'm happy to hear your thoughts.

Reply:
oooooh yeah because it's an extra now. I was thinking ms as being a default writable (requiring python-casacore too right?). but I think I agree with you, if this is a "full developer" yaml it should include pyuvata>=2.2.2

Reply:
ok, good. I updated it to 2.2.2